### ISSUE TITLE
invalid share on RTX 3060ti and 3070

### ISSUE THREAD

**Issue by sora251731** [PhoenixMinerDevTeam/PhoenixMiner#14]
Hi...Just want to ask about the issue on invalid share issue 

3060ti or 3070 oc when pheonixminer startup. Dag file load. invalid share will appear most of the time
3060ti or 3070 stock when pheonixminer startup. Dag file load. perfect. no invalid share

any update on this fix? since t-rex and nbminer does not have this kind of issue for now.
need some update on this. Thx

**Comment by deadpuck** [comment]
You should look into the readme, and documentation! They have a very nice list of different command line arguments that you can add to your mining.bat that allow you to not only overclock using Phoenix Miner but to also allow you to delay memory overclocks till after the DAG is loaded. 
-mcdag 1 -powlim -32 -cclock -500 -mclock +1370
Above are 4 arguments that I use on my 3060ti. the "-mcdag 1" Is telling Phoenix miner to delay memory overclocks. "-powlim -32" is the power limit overclock i set, 0 is 100% so -32 = 68% powerlimit.  cclock and mclock are of course your core clock and memory clock which is relatively standard. The only problem I've had after setting all these is if one of my cards crashes (my 1070 likes to crash if I'm using my computer to watch youtube videos sometimes) the miner doesn't undo overclocks (in normal circumstance if you close the miner it sets OC back to stock) and since the miner doesn't undo the overclock it causes the auto restart to be started with memory clock already on causing invalid shares. for most people this wouldn't be a problem but since i use my 1070 sometimes while mining on it. it likes to crash.

**Comment by sora251731** [comment]
> You should look into the readme, and documentation! They have a very nice list of different command line arguments that you can add to your mining.bat that allow you to not only overclock using Phoenix Miner but to also allow you to delay memory overclocks till after the DAG is loaded.
> -mcdag 1 -powlim -32 -cclock -500 -mclock +1370
> Above are 4 arguments that I use on my 3060ti. the "-mcdag 1" Is telling Phoenix miner to delay memory overclocks. "-powlim -32" is the power limit overclock i set, 0 is 100% so -32 = 68% powerlimit. cclock and mclock are of course your core clock and memory clock which is relatively standard. The only problem I've had after setting all these is if one of my cards crashes (my 1070 likes to crash if I'm using my computer to watch youtube videos sometimes) the miner doesn't undo overclocks (in normal circumstance if you close the miner it sets OC back to stock) and since the miner doesn't undo the overclock it causes the auto restart to be started with memory clock already on causing invalid shares. for most people this wouldn't be a problem but since i use my 1070 sometimes while mining on it. it likes to crash.

Eventhough if is set to delay....it is still not stable....once the Dag auto restart, you need to redo the whole process all over again
i did not set the overclock on my miner cause i m doing it at msi afterburner. but still after the delay of oc after dag is being load
in 24 hours of mining there is still some invalid shares and incorrect share exist...its makes me need to go over the process all over again. i m comparing with t-rex and nbminer because the miner has no issue on oc dag load...i don need to delay oc or worried about incorrect share

thats why i m asking the phoenix miner team, will they fix the issue on the future update?

currently using 4x 3080 2x 3070 2x 3060ti

**Comment by deadpuck** [comment]
> > You should look into the readme, and documentation! They have a very nice list of different command line arguments that you can add to your mining.bat that allow you to not only overclock using Phoenix Miner but to also allow you to delay memory overclocks till after the DAG is loaded.
> > -mcdag 1 -powlim -32 -cclock -500 -mclock +1370
> > Above are 4 arguments that I use on my 3060ti. the "-mcdag 1" Is telling Phoenix miner to delay memory overclocks. "-powlim -32" is the power limit overclock i set, 0 is 100% so -32 = 68% powerlimit. cclock and mclock are of course your core clock and memory clock which is relatively standard. The only problem I've had after setting all these is if one of my cards crashes (my 1070 likes to crash if I'm using my computer to watch youtube videos sometimes) the miner doesn't undo overclocks (in normal circumstance if you close the miner it sets OC back to stock) and since the miner doesn't undo the overclock it causes the auto restart to be started with memory clock already on causing invalid shares. for most people this wouldn't be a problem but since i use my 1070 sometimes while mining on it. it likes to crash.
> 
> Eventhough if is set to delay....it is still not stable....once the Dag auto restart, you need to redo the whole process all over again
> i did not set the overclock on my miner cause i m doing it at msi afterburner. but still after the delay of oc after dag is being load
> in 24 hours of mining there is still some invalid shares and incorrect share exist...its makes me need to go over the process all over again. i m comparing with t-rex and nbminer because the miner has no issue on oc dag load...i don need to delay oc or worried about incorrect share
> 
> thats why i m asking the phoenix miner team, will they fix the issue on the future update?
> 
> currently using 4x 3080 2x 3070 2x 3060ti

Ohh okay i haven't had the dag reload on me unless my GPU crashes so I've had similar experience between Trex and Phoenix, I've ran both miners for 72 hours consecutively without problems before as well. I do hope that they can fix your problem though in a future update!

**Comment by sora251731** [comment]
yup....its a hassle that the rig need to be monitor 12 to 24 hours...hope they fix it in the future update

**Comment by PhoenixMinerDevTeam** [comment]
> > You should look into the readme, and documentation! They have a very nice list of different command line arguments that you can add to your mining.bat that allow you to not only overclock using Phoenix Miner but to also allow you to delay memory overclocks till after the DAG is loaded.
> > -mcdag 1 -powlim -32 -cclock -500 -mclock +1370
> > Above are 4 arguments that I use on my 3060ti. the "-mcdag 1" Is telling Phoenix miner to delay memory overclocks. "-powlim -32" is the power limit overclock i set, 0 is 100% so -32 = 68% powerlimit. cclock and mclock are of course your core clock and memory clock which is relatively standard. The only problem I've had after setting all these is if one of my cards crashes (my 1070 likes to crash if I'm using my computer to watch youtube videos sometimes) the miner doesn't undo overclocks (in normal circumstance if you close the miner it sets OC back to stock) and since the miner doesn't undo the overclock it causes the auto restart to be started with memory clock already on causing invalid shares. for most people this wouldn't be a problem but since i use my 1070 sometimes while mining on it. it likes to crash.
> 
> Eventhough if is set to delay....it is still not stable....once the Dag auto restart, you need to redo the whole process all over again
> i did not set the overclock on my miner cause i m doing it at msi afterburner. but still after the delay of oc after dag is being load
> in 24 hours of mining there is still some invalid shares and incorrect share exist...its makes me need to go over the process all over again. i m comparing with t-rex and nbminer because the miner has no issue on oc dag load...i don need to delay oc or worried about incorrect share
> 
> thats why i m asking the phoenix miner team, will they fix the issue on the future update?
> 
> currently using 4x 3080 2x 3070 2x 3060ti

There is no way to make this work while using external program to overclock the GPU - we can't control MSI Afterburner in any way, and when we reset the clocks before the generation of the new DAG, nothing prevents Afterburner from overclocking the memory during the DAG generation. Your best bet is to stop using Afterburner for the overclock, and use only PhoenixMiner with the same settings, and with the option `-mcdag 1`

Note that you will need to run PhoenixMiner.exe as administrator to apply OC settings on Nvidia cards - this is requirement of all relatively recent Nvidia drivers.

**Comment by AuthenticAMD** [comment]
Wanted to comment that T-rex miner is able to generate a DAG while the cards are overclocked without causing invalid shares. How do they do it? I don't know. Maybe the generate the DAG by CPU and compare the has to the one generated by the GPU and if it doesn't match, it generates it again?
